packages/editor-ui/src/plugins/telemetry/index.ts

```diff
@@ -173,5 +196,6 @@ class Telemetry {
 		};
 		this.telemetry.loadJS();
 		this.telemetry.load(key, url, options);
+		this.flushPageEvents();
```

Comment:
how about moving this call to `init` method, after L66? so that page is called after the initial identify.

Reply:
updated